includes/class-wc-stripe-order-helper.php

```
@@ -652,6 +654,58 @@ public function update_stripe_mandate_id( ?WC_Order $order = null, string $manda
+	 *
+	 * @param string $mandate_id The Stripe mandate ID.
+	 * @return WC_Order|null The order, or null if not found.
+	 */
+	public function get_order_by_mandate_id( string $mandate_id ): ?WC_Order {
+		if ( empty( $mandate_id ) ) {
+			return null;
+		}
+
+		global $wpdb;
+
+		if ( WC_Stripe_Woo_Compat_Utils::is_custom_orders_table_enabled() ) {
```

PHPMD (2.15.0)

[error] Avoid using static access to class 'WC_Stripe_Woo_Compat_Utils' in method 'get_order_by_mandate_id'. (undefined)

(StaticAccess)